### PR TITLE
remove build status

### PR DESCRIPTION
**Pull request by nickdesaulniers**
Speculation:
It seems recently that github has been rate limiting requests to github.com workflow badge svg files from non-github.com URLs.

Sadly, this seems to include github.io pages, too, which smells like an oversight to me.

What this means in practice is that our homepage looks like hot dogshit full of broken links vs the github page.

Compare
https://clangbuiltlinux.github.io/
vs
https://github.com/ClangBuiltLinux/continuous-integration2/#readme

The latter is much better looking (no broken embedded images) than the former (many broken embedded images).

It's sad that this is necessary, but I think it will help the home page look more professional.  It will also be one less thing to update when we update the matrix using markdown-badges.py.

### CODE REVIEW

**Comment by JustinStitt** [comment]
Can you upload an artifact of what you're seeing, @nickdesaulniers ?

I see a pleasant home page on my end.

image

Granted, I don't check the home page very often -- Are you seeing problems only every once in awhile?

**Review by nathanchance — APPROVED** [review]
Unfortunate :( Should we still have some sort of callout, maybe something to the effect of "If you want to see the latest build status, go to the README of [our continuous-integration2 repository](https://github.com/ClangBuiltLinux/continuous-integration2)"? Regardless, this is obviously an improvement over the status quo so merge away.

File: readme.md [hunk 1]
```diff
@@ -2,40 +2,11 @@
...
-
-[![Check clang version](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/clang-version.yml/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions/workflows/clang-version.yml)
-
 ## Useful links
 
 - [Official Kernel Docs](https://www.kernel.org/doc/html/latest/kbuild/llvm.html)
 - [Issue tracker](https://github.com/ClangBuiltLinux/linux/issues)
+- [Build Status](https://github.com/ClangBuiltLinux/continuous-integration2/#readme)
```
Comment (nickdesaulniers) [inline readme.md:9]: note: we still retain a link, so "remove build status" is not accurate. More like "replace build status image grid in favor of link"

**Comment by JustinStitt** [comment]
I can't get the broken page to appear even with lots of refreshes and private tabs + clearing cache. If this is an issue for others though then this change is good.

**Comment by nickdesaulniers** [comment]
gah, the github.com page cuts off the content and is very difficult to scroll!

image

**Comment by JustinStitt** [comment]
Are there any large github projects with similar matrices that we can steal the formatting from?

**Comment by nickdesaulniers** [comment]
I have filed a bug against github support: https://support.github.com/ticket/personal/0/2313412 (probably not publicly accessible).

**Comment by JustinStitt** [comment]
Is this at all related to whatever the heck a webhook is: https://www.githubstatus.com/incidents/f7lz4nc72bvp

**Comment by kees** [comment]
Unfortunately, I think this is the right solution until github fixes their ACLs. I wonder if I could host something to cache the results? I could put it somewhere on .outflux.net ....

**Comment by JustinStitt** [comment]
> Unfortunately, I think this is the right solution until github fixes their ACLs. I wonder if I could host something to cache the results? I could put it somewhere on .outflux.net ....

I wonder if we can show some aggregate statistic like "Passing Builds 24/36" or something like that -- then also provide a link to the readme matrix.

**Comment by nathanchance** [comment]
Just as a data point, I see the same problem in Safari:

image

**Comment by nickdesaulniers** [comment]
@nathanchance 

> Just as a data point, I see the same problem in Safari:

Was that on clangbuiltlinux.github.io, or https://github.com/ClangBuiltLinux/continuous-integration2#readme ?

**Comment by nathanchance** [comment]
> @nathanchance
> 
> > Just as a data point, I see the same problem in Safari:
> 
> Was that on clangbuiltlinux.github.io, or https://github.com/ClangBuiltLinux/continuous-integration2#readme ?

clangbuiltlinux.github.io, continuous-integration2's has always worked for me.

**Comment by nickdesaulniers** [comment]
> I wonder if we can show some aggregate statistic like "Passing Builds 24/36" or something like that -- then also provide a link to the readme matrix.

Maybe worth researching shields.io. https://github.com/badges/shields

**Comment by nickdesaulniers** [comment]
> I have filed a bug against github support: https://support.github.com/ticket/personal/0/2313412 (probably not publicly accessible).

oh, I did receive a response back on aug 30:

> Hi Nick,
> Thank you for getting in touch with GitHub Support!
> As GitHub Pages are publicly accessible, GitHub does not process authentication when people visit one of these pages, meaning GitHub Pages sites are subject to the same rate limits as any other GitHub.com page when visited by an unauthenticated user.
> The result of this is the same as if you were to [view your repository page](https://github.com/ClangBuiltLinux/ClangBuiltLinux.github.io) while not logged in, some of the badges do not load as the rate limit is reached.
> As this is a site-wide rate limit we are unable to change the limit for your GitHub Pages site.
> You might consider breaking your website up into subdirectories divided by your current headers, this way each section would get a chance to load independently without hitting the rate limit.
> I hope this clears things up, please let me know if you have any further questions!

**Comment by JustinStitt** [comment]
Closing this as it was fixed by @kees 's efforts here: https://github.com/ClangBuiltLinux/continuous-integration2/pull/678

**Pull request closed without merging**